_meta/beat.yml

```diff
@@ -264,8 +264,11 @@ apm-server:
         # IPv6 addresses should always be defined as: `https://[2001:db8::1]:9200`.
         # hosts: ["localhost:9200"]
 
-        # Optional protocol and basic auth credentials.
+        # Protocol - either `http` (default) or `https`.
         #protocol: "https"
+
+        # Authentication credentials - either API key or username/password.
+        #api_key: "id:api_key"
```

Comment:
is this the base64 encoded version or really two strings with colon?

Reply:
It is really two strings with colon, the libbeat elasticsearch client takes care of the base64 encoding. 
We should be consistent with this behaviour also for sourcemaps, so I am changing the `elasticsearch/client` behavior accordingly. 
Unfortunately we currently do not have system tests for the case that an `apm-server.source_mapping.elasticsearch` is configured, created an issue for following up on that https://github.com/elastic/apm-server/issues/3166.

Reply:
great, thanks for clarifying